keyboards/40percentclub/tomato/keymaps/default/keymap.c

```diff
@@ -92,27 +92,12 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    * '-------------------------------------------------------------------------------'
    */
   LAYOUT_ortho_3x10
-    ( KC_CALC,KC_WSCH,KC_MAIL,KC_MYCM,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS
+    ( KC_CALC,KC_WSCH,KC_MAIL,KC_MYCM,_______,_______,_______,_______,_______,_______
     , RGB_TOG,RGB_MOD,RGB_HUI,RGB_HUD,KC_NO,  KC_NO,  RGB_SAI,RGB_SAD,RGB_VAI,RGB_VAD
```

Comment:
You may want to swap `KC_NO` for `XXXXXXX` here, since you've used the macro for `KC_TRNS` on this file as well.

Reply:
Resolved.